src/modules/Resources/Buckets.ts

```diff
@@ -127,6 +143,8 @@ class Buckets extends TagoIOModule<GenericModuleParams> {
    * ```
    * @param bucketID Bucket ID
    * @param optionsObj Request options
+   *
+   * @deprecated
```

Comment:
From this point forward, methods don't have a corresponding route on `Resource.devices` module so only added `@deprecated`.